src/types/sequence.rs

```diff
@@ -328,12 +316,25 @@ impl<'v> PyTryFrom<'v> for PySequence {
     }
 }
 
+fn extract_sequence<'s, T>(obj: &'s PyAny) -> PyResult<Vec<T>>
+where
+    T: FromPyObject<'s>,
+{
+    let seq = <PySequence as PyTryFrom>::try_from(obj)?;
+    let mut v = Vec::with_capacity(seq.len().unwrap_or(0) as usize);
+    for item in seq.iter()? {
+        v.push(item?.extract::<T>()?);
+    }
+    Ok(v)
+}
+
```

Comment:
This function having moved doesn't seem relevant, can you please put it back in the original spot?

Reply:
This was absolutely unintentional.